frontend_v2/src/app/components/challenge/challengeviewallsubmissions/challengeviewallsubmissions.component.html

```diff
@@ -228,6 +209,26 @@ <h5 class="fw-light">All Submissions</h5>
                     </strong>
                   </p>
                 </div>
+                <div class="element">
```

Comment:
please rename the css class to something relevant

Reply:
@Ram81, the name element is used across `challengviewsallsubmissions.components.html` and `challengsubmission.components.html` to represent a block with column flex property. All the class nomenclature in these files are revolving around `element` like elements, elements-detail, elements-status, and has been consistent all across. So should I change the name at all its occurrences and corresponding CSS across all the files or keep it as is?

Reply:
I see, let's not change it then